sample.tf

```diff
@@ -8,16 +8,26 @@ terraform {
   }
 }
 
-resource "project" "myproject" {
+resource "project_project" "myproject" {
   key = "myproj"
   display_name = "My Project"
-  description = "My Project"
+  description  = "My Project"
   admin_privileges {
-    manage_members = true
+    manage_members   = true
     manage_resources = true
-    index_resources = true
+    index_resources  = true
   }
-  max_storage_in_gigabytes = 10
+  max_storage_in_gigabytes   = 10
```

Comment:
I had had in mind that they can determine the unit because it's not hard to see petabytes being defined. It would make for a harder impl (but not much).

Reply:
I'm gonna keep it as is for now, but rename to `max_storage_in_gibabytes` so it's consistent with the API requirement and conversion we are doing.